app/Listeners/PsrWorkerErrorListener.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Listeners;
+
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use pushrbx\LumenRoadRunner\Events\LoopErrorOccurredEvent;
+
+class PsrWorkerErrorListener
+{
+    private \App\Exceptions\Handler $exceptionHandler;
+    private Logger $logger;
+
+    public function __construct()
+    {
+        $this->exceptionHandler = new \App\Exceptions\Handler;
+        $this->logger = new Logger('psr-worker');
+        $this->logger->pushHandler(new StreamHandler(storage_path().'/logs/psr-worker.log'), env('APP_DEBUG') ? Logger::DEBUG : Logger::WARNING);
+    }
+
+    public function handle(LoopErrorOccurredEvent $event): void
+    {
+        $this->logger->info("error logging...");
```

Comment:
Intentionally left in?

Reply:
copy pasted example code and accidentally left it in, not needed